src/main/java/org/terasology/staticCities/raster/standard/RoundPartRasterizer.java

```diff
@@ -48,14 +50,13 @@ public RoundPartRasterizer(BlockTheme theme) {
     @Override
     protected void raster(RasterTarget brush, RoundBuildingPart element, HeightMap heightMap) {
 
-        Circle area = element.getShape();
-
-        if (!area.intersects(brush.getAffectedArea())) {
+        Circlef area = element.getShape();
+        if (!CircleUtility.intersect(area, brush.getAffectedArea().getBounds(new Rectanglef()))) {
             return;
         }
 
-        Vector2i center = new Vector2i(area.getCenter(), RoundingMode.HALF_UP);
-        int radius = TeraMath.floorToInt(area.getRadius());
+        Vector2i center = new Vector2i(area.x, area.y, RoundingMode.HALF_UP);
+        int radius = TeraMath.floorToInt(area.r);
```

Comment:
Do we want to be able to access these directly instead of via getters?

Reply:
`Circlef` doesn't have a constant interface and the members are only directly accessible.